### PR TITLE
Add CCE to OCP rules

### PR DESCRIPTION
**Pull request by shawndwells**

### CODE REVIEW

File: applications/openshift/kubelet/kubelet_configure_tls_cert/rule.yml [hunk 1]
```diff
@@ -30,3 +33,4 @@ ocil: |-
     Verify that a certificate path is configured.
     <pre>$ sudo ls /etc/origin/node/certificates/kubelet-client-current.pem</pre>
     Verify that a client certificate is configured.
+
```
Comment (redhatrises) [inline applications/openshift/kubelet/kubelet_configure_tls_cert/rule.yml]: @shawndwells please remove this empty line that was added.

Comment (shawndwells) [reply applications/openshift/kubelet/kubelet_configure_tls_cert/rule.yml]: done

**Comment by shawndwells** [comment]
Ready to review. Did not add CCEs into api-server. Waiting until after @redhatrises pending PR to reduce any risk of merge conflict.

**Comment by jan-cerny** [comment]
@shawndwells LGTM, but there is a conflict with the other PR that has been merged.

**Pull request merged**